server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessagesMethodTest.java

```diff
@@ -214,7 +214,7 @@ public void getMessagesShouldReturnMessageWhenHtmlMessage() throws Exception {
             .body(ARGUMENTS + ".list[0].subject", equalTo("my test subject"))
             .body(ARGUMENTS + ".list[0].htmlBody", equalTo("This is a <b>HTML</b> mail"))
             .body(ARGUMENTS + ".list[0].isUnread", equalTo(true))
-            .body(ARGUMENTS + ".list[0].preview", equalTo("This is a <b>HTML</b> mail"))
+            .body(ARGUMENTS + ".list[0].preview", equalTo("This is a HTML mail"))
             .body(ARGUMENTS + ".list[0].headers", equalTo(ImmutableMap.of("content-type", "text/html", "subject", "my test subject")))
```

Comment:
I would like to see a test were text is here

If the content type is text, HTML tag should not be removed IMO. Image I send a text mail like this : 

```
Content-type : text/plain

Little cousin, if you want to put some bold content, do it like this : <b>text in bold</b>
```